src/pybind/mgr/dashboard/frontend/cypress/integration/nfs/nfs.e2e-spec.ts

```diff
@@ -1,16 +0,0 @@
-import { NfsPageHelper } from './nfs.po';
-
-describe('Nfs page', () => {
-  const nfs = new NfsPageHelper();
-
-  beforeEach(() => {
-    cy.login();
-    nfs.navigateTo();
-  });
-
```

Comment:
nfs-e2e-spec was related to checking a 'NFS' breadcrumb on clicking the NFS tab. Since we are showing a generic error page on clicking this tab so it was not feasible to show the breadcrumb now.